Apps/Src/Task_VoltTempMonitor.c

```diff
@@ -111,13 +111,23 @@ void Task_VoltTempMonitor(void *p_arg) {
         }
         bool charge_enable = true;
         //Check if car should be allowed to charge or not
-        for(uint8_t k = 0; k < NUM_BATTERY_MODULES; k++){
-            uint32_t temp = Temperature_GetModuleTemperature(k);
-            if(temp > MAX_CHARGE_TEMPERATURE_LIMIT && temp < MAX_DISCHARGE_TEMPERATURE_LIMIT){
-                //suggest that the battery should not be charged
-                charge_enable = false;
+        for (uint8_t board = 0; board < NUM_MINIONS; board++) {
```

Comment:
Is the purpose of breaking the single for loop into a nested for loop just to make it clearer what the actual system (ie 4 minions, 8 modules for each except for 1 with 7) is like, or is there another purpose?

Reply:
Yeah, I think it's easier to read and `Temperature_GetSingleTempSensor()` takes `board` and `sensor` as inputs, so I think it's a little bit nicer if we already have those values from the nested loop variables